apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts

```diff
@@ -24,8 +24,9 @@ import { Utilities } from '../../utilities/Utilities';
 import { RushConstants } from '../../logic/RushConstants';
 import { EnvironmentVariableNames } from '../../api/EnvironmentConfiguration';
 import { LastLinkFlag, LastLinkFlagFactory } from '../../api/LastLinkFlag';
-import { IRushConfigurationProjectJson } from '../../api/RushConfigurationProject';
+import { IRushConfigurationProjectJson, RushConfigurationProject } from '../../api/RushConfigurationProject';
 import { BuildCacheConfiguration } from '../../api/BuildCacheConfiguration';
+import * as Selection from '../../logic/Selection';
```

Comment:
Make `Selection` a class.  

Using `import * as Selection` has a number of downsides:
- When I search our code for `Selection.union`, I may miss some places where developers inconsistently chose not to use `import * as Selection`.
- If the class gets renamed, people can forget to update some of the imports, and the compiler won't catch that
- As a general pattern `import * as Selection` only works for internal APIs; API Extractor won't know how to document it as a namespace because it is compositionally constructed

Reply:
Switched to using a (fake) class.